server/parameters/ServerSubcommand.java

```diff
@@ -56,9 +56,9 @@ public Export asExport() {
         private final boolean isDebug;
         private final boolean isHelp;
         private final boolean isVersion;
-        private final Config config;
```

Comment:
All the changes in this file are from prefixing `Config` class with the word `Core`.